Formula/weighttp.rb

```diff
@@ -19,16 +19,20 @@ class Weighttp < Formula
     sha256 cellar: :any, yosemite:       "e83c9f99b524b57ba31571dc673ab6d2d2a5e38a5374ce45130f11a51c063662"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "libev"
 
   def install
-    system "./waf", "configure"
-    system "./waf", "build"
-    bin.install "build/default/weighttp"
+    system "./autogen.sh"
```

Comment:
Add `system "autoupdate"` before this line and it should work (at least it did for me in local testing).

Reply:
thanks for the pointer!